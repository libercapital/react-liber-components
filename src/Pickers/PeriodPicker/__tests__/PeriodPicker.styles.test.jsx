import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';
import {
  Content, MonthPickerContainer,
} from '../styles';

describe('Testing MonthPicker Styles', () => {
  configure({ adapter: new Adapter() });
  it('Month Should render correctly', () => {
    const wrapper = shallow(<Content />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
  it('Month Should render correctly ', () => {
    const wrapper = shallow(<MonthPickerContainer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
  it('Month Should render correctly selected with show', () => {
    const wrapper = shallow(<MonthPickerContainer show />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
