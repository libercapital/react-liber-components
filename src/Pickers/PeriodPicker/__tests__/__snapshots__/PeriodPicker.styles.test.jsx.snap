// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing MonthPicker Styles Month Should render correctly  1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "styles__MonthPickerContainer-sc-59to88-1",
        "isStatic": false,
        "rules": Array [
          "position:absolute;overflow:hidden;top:0px;right:0px;width:270px;display:",
          [Function],
          ";background-color:#fafafa;box-shadow:0 2px 6px 0 rgba(0,0,0,0.1);",
        ],
      },
      "displayName": "styles__MonthPickerContainer",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "styles__MonthPickerContainer-sc-59to88-1",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;

exports[`Testing MonthPicker Styles Month Should render correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "styles__Content-sc-59to88-0",
        "isStatic": false,
        "rules": Array [
          "position:relative;",
        ],
      },
      "displayName": "styles__Content",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "styles__Content-sc-59to88-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;

exports[`Testing MonthPicker Styles Month Should render correctly selected with show 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "styles__MonthPickerContainer-sc-59to88-1",
        "isStatic": false,
        "rules": Array [
          "position:absolute;overflow:hidden;top:0px;right:0px;width:270px;display:",
          [Function],
          ";background-color:#fafafa;box-shadow:0 2px 6px 0 rgba(0,0,0,0.1);",
        ],
      },
      "displayName": "styles__MonthPickerContainer",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "styles__MonthPickerContainer-sc-59to88-1",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
  show={true}
/>
`;
